src/buckets.spec.ts

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable import/first */
+;(global as any).WebSocket = require('isomorphic-ws')
+
+import fs from 'fs'
+import path from 'path'
+import { isBrowser, isNode } from 'browser-or-node'
+import { ThreadID } from '@textile/threads-id'
+import { expect } from 'chai'
+import { Client } from '@textile/threads-client'
+import { InitReply } from '@textile/buckets-grpc/buckets_pb'
+import { Context } from './context'
+import { Buckets } from './buckets'
+import { signUp } from './utils'
+
+// Settings for localhost development and testing
+const addrApiurl = 'http://127.0.0.1:3007'
+const addrGatewayUrl = 'http://127.0.0.1:8006'
+const wrongError = new Error('wrong error!')
+const sessionSecret = 'textilesession'
+
+describe('Buckets...', () => {
+  let ctx: Context = new Context(addrApiurl, undefined)
+  const client = new Buckets(ctx)
+  let buck: InitReply.AsObject
+  let fileSize: number
+  before(async () => {
+    const user = await signUp(ctx, addrGatewayUrl, sessionSecret)
```

Comment:
just walking through the js todo example wiring this up. 

how should another app use this? should i import `pb` and use the `pb.signup..` method directly in the todo app or is there another way?

Reply:
No, you shouldn't use this at all. You should do this part all from the cli.